test/e2e/framework/ginkgowrapper.go

```diff
@@ -67,6 +125,301 @@ func AnnotatedLocationWithOffset(annotation string, offset int) types.CodeLocati
 
 // ConformanceIt is wrapper function for ginkgo It.  Adds "[Conformance]" tag and makes static analysis easier.
 func ConformanceIt(text string, args ...interface{}) bool {
-	args = append(args, ginkgo.Offset(1))
-	return ginkgo.It(text+" [Conformance]", args...)
+	args = append(args, ginkgo.Offset(1), WithConformance())
+	return It(text, args...)
+}
+
+// It is a wrapper around [ginkgo.It] which supports framework With* labels as
+// optional arguments in addition to those already supported by ginkgo itself,
+// like [ginkgo.Label] and [gingko.Offset].
+//
+// Text and arguments may be mixed. The final text is a concatenation
+// of the text arguments and special tags from the With functions.
+func It(text string, args ...interface{}) bool {
+	return registerInSuite(ginkgo.It, text, args)
+}
+
+// It is a shorthand for the corresponding package function.
```

Comment:
naive question, why do we want the shorthand?

Reply:
It makes lines quite a bit shorter. That's really all there is to it. For the difference, see https://github.com/kubernetes/kubernetes/pull/112894/commits/53c831d6a5708e927934c1a419237abe8d69383b.

I find it much nicer with the shorthand, but can also live with the long form.

Reply:
no strong opinion, just that I can see how we may  end with a mix of both forms